spec/openapi.yaml

```diff
@@ -53,29 +149,213 @@ paths:
+                          'category': 'Tutorial',
+                          'created_at': '2019-07-21 15:41:19',
+                          'downvotes': 0,
+                          'id': 2137,
+                          'languages': [],
+                          'last_updated': '2019-07-22 16:05:53',
+                          'name': 'Some, but not ALL of the knowledge',
+                          'notes': null,
+                          'paid': false,
+                          'times_clicked': 0,
+                          'upvotes': 1,
+                          'url': 'http://www.test.com'
+                      },
+                      'status': 'ok',
+                      'status_code': 200
+                    }
+      security:
+      - ApiKeyAuth: []
+  /resources/{id}/click:
+    put:
+      summary: UPDATE RESOURCE VISITS BY ID
+      description:
+        Increments the `times_clicked` property of a resource 
+      parameters:
+      - name: id
+        in: path
+        description: Resource ID
+        required: true
+        style: simple
+        explode: false
+        schema:
+          type: integer
+          format: int64
+      responses:
+        200:
+          description: Updated resource details
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Resource'
               examples:
                 0:
                   value: |
-                    {"apiVersion":"1.0","data":{"category":"Books","created_at":"2019-06-15 17:55:32","downvotes":0,"id":3,"languages":[],"last_updated":"","name":"Free Tech Books","notes":"Focuses on general computer science concepts rather than a specific language","paid":false,"times_clicked":0,"upvotes":0,"url":"http://www.freetechbooks.com/"},"status":"ok","status_code":200}
+                    {
+                      'apiVersion': '1.0',
+                      'data': {
+                          'category': 'Tutorial',
+                          'created_at': '2019-07-21 15:41:19',
+                          'downvotes': 0,
+                          'id': 2137,
+                          'languages': [],
+                          'last_updated': '2019-07-22 16:05:53',
+                          'name': 'Some, but not ALL of the knowledge',
+                          'notes': null,
+                          'paid': false,
+                          'times_clicked': 3,
+                          'upvotes': 1,
+                          'url': 'http://www.test.com'
+                      },
+                      'status': 'ok',
+                      'status_code': 200
+                    }
+        404:
+          $ref: '#/components/responses/NotFound'
   /resources:
     get:
-      description: Gets list of all available resource entries
+      summary: RETRIEVE RESOURCES
+      description: Gets list of all available resource entries or filtered entries based on provided criteria.
+      parameters:
```

Comment:
This is missing documentation on the `updated_after` parameter. Description should be 
```
Filters resources that were updated after the specified date. To get all resources updated after July 17, 2019, use `/resources?updated_after=2019-07-17`
```
Type should be `date` or `datetime`, I'm not sure what OpenAPI supports. If either of those work, it can be `date`.

Everything else regarding this param should be pretty obvious so I think you can take it from there.